frontend/kesaseteli/employer/src/pages/_app.tsx

```diff
@@ -22,17 +22,9 @@ import { ThemeProvider } from 'styled-components';
 
 import { BackendEndPoints, getBackendDomain } from '../backend-api/backend-api';
 
-// Sentry logger
```

Comment:
I wonder if we just still keep the react-query logger here, or didnt it work?
https://react-query.tanstack.com/reference/setLogger